toolkit/tools/imagecustomizer/docs/configuration.md

```diff
@@ -346,49 +346,86 @@ Specifies the configuration for the generated ISO media.
 
 Specifies the configuration for overlay filesystem.
 
-- `lowerDir`: This directory acts as the read-only layer in the overlay
-  filesystem. It contains the base files and directories which will be overlaid
-  by the upperDir. Changes to the overlay filesystem do not affect the contents
-  of lowerDir.
+- `lowerDir`: (Required)
```

Comment:
In the documentation for the other types, the subfields are headers, not a list.